include/mmx/OCL_VDF.h

```diff
@@ -31,6 +31,8 @@ class OCL_VDF {
 
 	void verify(std::shared_ptr<const ProofOfTime> proof, const uint32_t chain);
 
+	void compute_point(std::vector<hash_t>* point_p, std::vector<uint32_t>* num_iters_p, const size_t start_p, const size_t num_p);
```

Comment:
I would use pointers only when the argument is optional, otherwise I use reference with `&`.

Reply:
Will fix, depending on new variant of load balance.